skaffold-operator.yaml

```
@@ -899,3 +902,214 @@ profiles:
           replicaCount: 1
           config.leaderElection.leaderElect: false
           config.server.healthProbes.enable: false
+---
+apiVersion: skaffold/v4beta7
+kind: Config
+metadata:
+  name: provider-local
+build:
+  tagPolicy:
+    customTemplate:
+      template: "{{.version}}-{{.sha}}"
+      components:
+        - name: version
+          envTemplate:
+            template: "{{.GARDENER_VERSION}}"
+        - name: sha
+          gitCommit:
+            variant: AbbrevCommitSha
+  artifacts:
+    - image: europe-docker.pkg.dev/gardener-project/releases/gardener/extensions/provider-local
```

```suggestion
    - image: local-skaffold/gardener-extension-provider-local
```

[Ref](https://github.com/gardener/gardener/blob/cba55148ed937263289ce52ffa839e02573bc833/skaffold.yaml#L599)